### ISSUE TITLE
C# Quick starts fail on Safari

### ISSUE THREAD

**Issue by BillWagner** [dotnet/try#13]
I tried to run the first hello World quick start using Safari on Mac.  It consistently fails.

One try had this request ID: `Request-Id: |29027195-4d151a5e318c6cd6.1.2.106.1.`

The failures are consistent for any of the examples.

/cc @LadyNaggaga @piotrpMSFT

**Comment by jonsequitur** [comment]
@BillWagner, does the issue persist if you try an incognito browser session?

**Comment by BillWagner** [comment]
@jonsequitur Yes.  Here's the request ID from the incognito window: Request-Id: |9e01c057-40049698d4f447e0.1.2.132.1.

**Comment by jonsequitur** [comment]
More repro details:

This only affects a previously-uncookied session on Safari at e.g. https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world#step-1. It doesn't occur at https://try.dot.net. If you run a snippet at https://try.dot.net, then return to the Quickstart, the Quickstart will now work.

**Comment by BillWagner** [comment]
One note: you can substitute any of the quick starts for the hello world above. It's not specific to that page.

**Comment by TheRealPiotrP** [comment]
The workaround for thus is to visit try.dot.net directly before visiting docs. There is a privacy feature in safari that prevents iFramed try.dot.net from setting cookies like anti forgery, breaking the experience. We are looking into workarounds...
